src/pages/my_page/user_profile/MyProfile.tsx

```diff
@@ -296,20 +297,41 @@ export const MyProfile = () => {
                     </Field>
                   </Grid>
                 </Grid>
-
-                <Box
-                  sx={{
-                    gridArea: 'profile-picture',
-                    display: 'flex',
-                    flexDirection: 'column',
-                    alignItems: 'center',
-                    mb: { xs: '1rem', lg: 0 },
-                  }}>
-                  <Typography variant="h3" sx={{ mb: '1rem' }}>
-                    {t('my_page.my_profile.profile_picture')}
-                  </Typography>
-                  <ProfilePictureUploader personId={personId} />
-                </Box>
+                <FormBox sx={{ mb: '1rem' }}>
+                  <Box
+                    sx={{
+                      gridArea: 'profile-picture',
+                      display: 'flex',
+                      flexDirection: 'column',
+                      alignItems: 'center',
+                      mb: { xs: '1rem', lg: 0 },
+                    }}>
+                    <Typography variant="h3" sx={{ mb: '1rem' }}>
+                      {t('my_page.my_profile.profile_picture')}
+                    </Typography>
+                    <Trans
+                      i18nKey="my_page.my_profile.upload_is_not_mandatory"
+                      components={[
+                        <Typography
+                          key="1"
+                          paragraph
+                          sx={{ mb: '0.25rem', textAlign: 'center', fontStyle: 'italic' }}
```

Comment:
`paragraph`-prop setter ganske stor margin bottom, som igjen overstyres her. Ser ut som det du egentlig vil ha er `gutterBottom`?
```suggestion
                          gutterBottom
                          sx={{ textAlign: 'center', fontStyle: 'italic' }}
```